packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -266,6 +266,15 @@ export interface IContextualMenuProps extends IBaseProps<IContextualMenu>, IWith
    * This will only result in different behavior when `shouldFocusOnMount = false`.
    */
   delayUpdateFocusOnHover?: boolean;
+
+  /**
+   * Called when the component is unmounting, and focus needs to be restored.
+   * Argument passed down contains two variables, the element that the underlying
+   * popup believes focus should go to * and whether or not the popup currently
+   * contains focus. If this is provided, focus will not be restored automatically,
```

Comment:
Nit - looks like line-wrapping stuck a `*` on line 273. Would also clarify definition of `this`, maybe say "if this prop is passed" ?